addons/sourcemod/scripting/ttt/ttt_martydom.sp

```diff
@@ -0,0 +1,174 @@
+#include <sourcemod>
+#include <sdkhooks>
+#include <sdktools>
+#include <cstrike>
+#include <ttt_shop>
+#include <ttt>
+#include <config_loader>
+#include <multicolors>
+
+#pragma newdecls required
+
+#define SHORT_NAME "md"
+#define SHORT_NAME_T "md_t"
+#define SHORT_NAME_D "md_d"
+
+#define PLUGIN_NAME TTT_PLUGIN_NAME ... " - Items: Martydom"
+
+char g_sLongName[PLATFORM_MAX_PATH] = "Martydom";
+char g_sPluginTag[PLATFORM_MAX_PATH];
+char g_sConfigFile[PLATFORM_MAX_PATH] = "";
+
+float g_fDeathPos[MAXPLAYERS+1][3];
+float g_fGrenadeDelay = 0.25;
+float g_fDamage = 99.0;
+float g_fExplosionSize = 350.0;
+
+int g_iIPrice = 0;
+int g_iIPrio = 0;
+
+int g_iDPrice = 0;
+int g_iDPrio = 0;
+
+int g_iTPrice = 0;
+int g_iTPrio = 0;
+
+bool g_bHasMartydom[MAXPLAYERS+1];
+
+/* Drops a live high explosive grenade on a players body */
+
+public Plugin myinfo = 
+{
+	name = PLUGIN_NAME, 
+	author = TTT_PLUGIN_AUTHOR,
+	description = TTT_PLUGIN_DESCRIPTION, 
+	version = TTT_PLUGIN_VERSION, 
+	url = TTT_PLUGIN_URL
+};
+
+public void OnPluginStart() {
+
+	TTT_IsGameCSGO();
+	
+	LoadTranslations("ttt.phrases");
+
+	BuildPath(Path_SM, g_sConfigFile, sizeof(g_sConfigFile), "configs/ttt/config.cfg");
+	Config_Setup("TTT", g_sConfigFile);
+
+	Config_LoadString("ttt_plugin_tag", "{orchid}[{green}T{darkred}T{blue}T{orchid}]{lightgreen} %T", "The prefix used in all plugin messages (DO NOT DELETE '%T')", g_sPluginTag, sizeof(g_sPluginTag));
+
+	Config_Done();
+
+	BuildPath(Path_SM, g_sConfigFile, sizeof(g_sConfigFile), "configs/ttt/martydom.cfg");
+	Config_Setup("TTT-Martydom", g_sConfigFile);
+
+	Config_LoadString("martydom_name", "Martydom", "The name of this item in the shop.", g_sLongName, sizeof(g_sLongName));
+	
+	g_iIPrice = Config_LoadInt("md_i_price", 2000, "The amount of credits martydom costs as traitor. 0 to disable.");	
+	//g_iTPrice = Config_LoadInt("md_t_price", 2000, "The amount of credits martydom costs as traitor (Traitor). 0 to disable.");
```

Comment:
Why you comment this and use a fixed price for all roles? Maybe some communities wants different prices per role